eclair-node/pom.xml

```
@@ -49,25 +49,25 @@
                 </configuration>
             </plugin>
             <plugin>
-                <groupId>com.github.chrisdchristo</groupId>
-                <artifactId>capsule-maven-plugin</artifactId>
+                <artifactId>maven-assembly-plugin</artifactId>
+                <version>3.2.0</version>
+                <configuration>
+                    <finalName>${project.name}-${project.version}-${git.commit.id.abbrev}</finalName>
+                    <descriptors>
+                        <descriptor>modules/eclair-node.xml</descriptor>
```

Nit, could also be `${project.name}.xml` but I don't see the reason to provide too much cardinality.

```suggestion
                        <descriptor>modules/assembly.xml</descriptor>
```